pos_lot_selection/static/src/xml/LotSelectorPopup.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 Camptocamp SA
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+<templates id="template" xml:space="preserve">
+
+    <t t-inherit="point_of_sale.EditListPopup" t-inherit-mode="extension">
+
+        <xpath expr="//t[@t-foreach='state.array']" position="replace">
+            <t t-foreach="state.array" t-as="item" t-key="item._id">
+                <EditListInput item="item" lots="props.lots" />
```

Comment:
Shouldn't we do this only if `props.lots` is defined? To avoid messing with possible other usages of `EditListPopup`